microraiden/requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements.txt
 pytest
-ethereum-tester
+eth-tester
```

Comment:
Please revert this change. There are some compatibility issues in the ethereum libraries we are using and we need to fix these before we can use new `eth-tester` and others.